concordium-consensus/src/Concordium/Scheduler/Environment.hs

```diff
@@ -61,9 +64,17 @@ class (Monad m) => StaticInformation m where
   getAccountCreationLimit :: m CredentialsPerBlockLimit
 
 -- |Information needed to execute transactions in the form that is easy to use.
-class (Monad m, StaticInformation m, CanRecordFootprint (Footprint (ATIStorage m)), AccountOperations m, MonadLogger m, IsProtocolVersion pv)
+class (Monad m, StaticInformation m, TVer.TransactionVerifier m, CanRecordFootprint (Footprint (ATIStorage m)), AccountOperations m, MonadLogger m, IsProtocolVersion pv)
     => SchedulerMonad pv m | m -> pv where
 
+  -- |Insert an entry to the 'Cache'.
+  -- The first parameter, @TransactionHash@, is the key of the entry.
+  -- The second parameter, @CacheableVerificationResult@, is the value of the entry.
+  insertTransactionVerificationResult :: TransactionHash -> CacheableVerificationResult -> m ()
```

Comment:
I can't see where this is used. What is its purpose?

Reply:
It serves no purpose anymore. So I'll remove it.